src/test/java/org/sqlite/CollationTest.java

```diff
@@ -0,0 +1,165 @@
+package org.sqlite;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.Collator;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests User Defined Collations.
+ */
+public class CollationTest {
+    private Connection conn;
+    private Statement stat;
+
+    private static String valStr1 = "";
+    private static String valStr2 = "";
+
+    @Before
+    public void connect() throws Exception {
+        conn = DriverManager.getConnection("jdbc:sqlite:");
+        stat = conn.createStatement();
+    }
+
+    @After
+    public void close() throws SQLException {
+        stat.close();
+        conn.close();
+    }
+
+    @Test
+    public void reverseCollation() throws SQLException {
```

Comment:
Good exmaple ;)

Reply:
Inspired by the Python Sqlite [doc](https://docs.python.org/3/library/sqlite3.html#sqlite3.Connection.create_collation) ;)